analyzers/src/SonarAnalyzer.Common/DiagnosticAnalyzer/SonarDiagnosticAnalyzer.cs

```diff
@@ -24,8 +24,14 @@ namespace SonarAnalyzer.Helpers
 {
     public abstract class SonarDiagnosticAnalyzer : DiagnosticAnalyzer
     {
+        protected virtual bool EnableConcurrentExecution { get; } = true;
```

Comment:
I would do it this way, because this way it is only settable to false via override.

```suggestion
        protected virtual bool EnableConcurrentExecution => true;
```